ekncontent/docs/reference/ekncontent/eos-knowledge-content-sections.txt

```diff
@@ -137,16 +137,18 @@ EkncQueryObjectMode
 EkncQueryObjectMatch
 EkncQueryObjectSort
 EkncQueryObjectOrder
+eknc_query_object_get_corrected_query_string
 eknc_query_object_get_cutoff
 eknc_query_object_get_excluded_ids
 eknc_query_object_get_excluded_tags
+eknc_query_object_get_filter_string
+eknc_query_object_get_filter_out_string
 eknc_query_object_get_ids
 eknc_query_object_get_query_string
```

Comment:
`eknc_query_object_get_query` is missing from this list, the gtk-doc test fails.

Reply:
Fixed, and `make check` now passes inside ekncontent.